docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2006,10 +2006,10 @@ and Docker images report:
     HTTP/1.1 200 OK
     Content-Type: application/json
 
-    {"status": "create", "id": "dfdf82bd3881","from": "ubuntu:latest", "time":1374067924}
-    {"status": "start", "id": "dfdf82bd3881","from": "ubuntu:latest", "time":1374067924}
-    {"status": "stop", "id": "dfdf82bd3881","from": "ubuntu:latest", "time":1374067966}
-    {"status": "destroy", "id": "dfdf82bd3881","from": "ubuntu:latest", "time":1374067970}
+    {"status":"pull","id":"busybox:latest","time":1442421700,"timeNano":1442421700598988358}
+    {"status":"create","id":"5745704abe9caa57898ed7c1e2f85b73b00e2b128a2d681c74bf4c46d00a717d","from":"busybox","time":1442421716,"timeNano":1442421716853979870}
+    {"status":"attach","id":"5745704abe9caa57898ed7c1e2f85b73b00e2b128a2d681c74bf4c46d00a717d","from":"busybox","time":1442421716,"timeNano":1442421716894759198}
+    {"status":"start","id":"5745704abe9caa57898ed7c1e2f85b73b00e2b128a2d681c74bf4c46d00a717d","from":"busybox","time":1442421716,"timeNano":1442421716983607193}
```

Comment:
Hm, bit worried that the useful information will now be hidden behind a scrollbar.

Thinking how to best solve this;
"cheat", and use a short ID in the output (did this ever return a short-ID as in the old output?), or
reformat the JSON;

```JSON
{
    "status": "create",
    "id": "5745704abe9caa57898ed7c1e2f85b73b00e2b128a2d681c74bf4c46d00a717d",
    "from": "busybox",
    "time": 1442421716,
    "timeNano": 1442421716853979870
}
```

If we *do* reformat the JSON, we should add a note (similar to [what's here](https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.21.md#get-an-archive-of-a-filesystem-resource-in-a-container), i.e. " (whitespace added for readability)"